ncm-metaconfig/src/main/metaconfig/beats/pan/schema_6.3.pan

```diff
@@ -128,6 +128,14 @@ type beats_logging = {
     'level' ? string with match(SELF, '^(critical|error|warning|info|debug)$')
 };
 
+@documentation{
+    Secomp settings for RHEL with Linux >3.16
+    (see https://www.elastic.co/guide/en/beats/filebeat/6.8/linux-seccomp.html)
+}
+type beats_seccomp = {
+    'default_action' : string
```

Comment:
Would a choice type make more sense here to validate the entry supplied?

Reply:
I think this is added as an optional workaround for users in particular sys admins on a system with more recent version of Linux. Thus, I don't think it is necessary to enforce/validate the value.